ftl-build/alpine/Dockerfile

```diff
@@ -68,7 +68,7 @@ RUN curl -sSL https://ftl.pi-hole.net/libraries/readline-${readlineversion}.tar.
 
 RUN curl -sSL https://ftl.pi-hole.net/libraries/nettle-${nettleversion}.tar.gz | tar -xz \
     && cd nettle-${nettleversion} \
-    && ./configure --enable-static --disable-shared \
+    && ./configure --enable-static --disable-shared --disable-openssl --disable-mini-gmp -disable-gcov --disable-documentation \
     && make -j $(nproc) install \
```

Comment:
I see you got some issues building ftl againt arm/v5. Cross compilation would speed up the build (a lot) as it will not use emulation (QEMU) at all. You can look at some examples using [xx](https://github.com/tonistiigi/xx) for setting this up if you want or I can have a look in follow-up:
* https://github.com/crazy-max/docker-msmtpd/blob/62fa294ca60e92e116b9b85d888dc54bc3f9139c/Dockerfile#L7-L27
* https://github.com/crazy-max/docker-7zip/blob/8f719b2ce3074818119cc19f2b16de5177bf0ad3/Dockerfile#L20-L27

Reply:
Thanks. Something to speed up the builds would be awesome!

What I want to achieve here is basically three-fold:

Firstly, I want to simplify out current tool chain of using completely different cross compile steps for each and every architecture so my initial idea was to set up one Alpine Dockerfile that handles compiling all the dependencies (libidn. libnettle, ...) because we often use features from the latest versions before they become available in the OS. It'd also have given us the benefit of all-static batteries-included binaries that will just run on this processor, irrespective how old (think Debian Wheezy) the OS-provided `libc` is.

Secondly, it'd be nice to continue supporting the whole bunch of ARM platforms that still exist in reality, so down to v4 would be the best. Since I found Alpine's minimum processor is armv6KZ, I added a second set of builders into the game: Debian. To avoid aforementioned `libc` incompatibilities best, we should use the oldest Debian that is still alive. Now the problems with this idea are: 1. `buster` dropped armv4, so we use `stretch` for this for now and will drop armv4 later. 2. `debian:stretch` compiling for armv6 actually produces an `armv4` binary. `debian:buster` is not even available for `armv6` (and I don't know why) so `debian:bullseye` would really be the only option even if the generated binary may not be runable on Debian Buster (we will have to try this!).

The remaining error is simple and I just haven't had time to fix it but a follow-up using some easier cross-compilation than what we had before would be awesome!